instruqt-tracks/terraform-basics-aws/track.yml

```diff
@@ -0,0 +1,472 @@
+slug: terraform-basics-aws
+id: 3grwcipacthi
+type: track
+title: Terraform Basics on AWS
+teaser: Learn how to begin provisioning infrastructure on AWS with Terraform.
+description: |-
+  Terraform is the world's most popular open source provisioning tool. Thousands of users build millions of containers, virtual machines, and other infrastructure with Terraform every single day. Terraform is easy to learn and fun to work with.
+
+  We invite you to join the ranks of the Terraform community today!
+
+  This track will introduce you to Terraform open source on AWS. You'll learn  terraform setup, basic commands and integration with AWS.
+icon: https://storage.googleapis.com/instruqt-hashicorp-tracks/logo/terraform.png
+tags:
+- terraform
+- aws
+- basics
+owner: hashicorp
+developers:
+- go@hashicorp.com
+private: true
+published: true
+challenges:
+- slug: setup-our-environment
+  id: xywjxcqb096l
+  type: challenge
+  title: "\U0001F3E1 Moving in - Set Up Your Workspace"
+  teaser: Configure your code editor with the HCL extension and open a workspace.
+  assignment: |-
+    **Goal**: Validate and configure your working environment.
+    **Estimated Time**: 2-3 minutes.
+
+    What you see on screen now will be your working environment for the rest of this training.
+
+    You'll notice that there are a few tabs along the top of the frame. The bright orange one will take you back to the Instruqt home page - try not to click it!
+
+    The next tab along should be active, and is your interactive shell. This is where you will be interacting with the Terraform CLI.
+
+    - In your shell, type `echo "hello world"`.
+
+    Finally, you have the Code Editor. This is where you will create and modify Terraform code.
+
+    1. Click on the Code Editor tab to switch context.
+
+    2. Select `File > Open...`.
```

Comment:
This should read `File > Open Folder`

![image](https://user-images.githubusercontent.com/403332/69433088-9047b100-0d00-11ea-8693-e7b6273cedd7.png)

Reply:
![image](https://user-images.githubusercontent.com/7535071/69841755-974c6180-12b4-11ea-92d8-a24d87564b68.png)
I changed this because of what I see on my machine. I suspect Instruqt is identifying the OS you come in on and presenting the respective VSCode menus. I'll change this to "File > Open" on Mac, or "File > Open Folder" on PC.